src/main/kotlin/no/nav/klage/oppgave/clients/OppgaveClient.kt

```diff
@@ -60,6 +60,28 @@ class OppgaveClient(
                 .block() ?: throw OppgaveClientException("Oppgaver could not be fetched")
         }
 
+    @Retryable
+    fun fetchAllKlageOppgaverForSYKBasedOnDate(includeFrom: LocalDate?, offset: Int, limit: Int = FETCH_LIMIT) =
+        logTimingAndWebClientResponseException("getOppgaver ($offset)") {
+            oppgaveWebClient.get()
+                .uri { uriBuilder ->
+                    uriBuilder.queryParam("statuskategori", Statuskategori.AAPEN)
+                    uriBuilder.queryParam("limit", limit)
+                    uriBuilder.queryParam("offset", offset)
+                    uriBuilder.queryParam("behandlingstype", BEHANDLINGSTYPE_KLAGE)
+                    uriBuilder.queryParam("tema", "SYK")
```

Comment:
Burde denne parameteriseres og være en del av request? Vi skal kanskje pushe in andre ytelser senere?

Reply:
I'm on it :)